Source/WebInspectorUI/UserInterface/Models/RecordingAction.js

```diff
@@ -115,6 +115,39 @@ WI.RecordingAction = class RecordingAction extends WI.Object
         return typeof propertyDescriptor.value === "function";
     }
 
+    static bitfieldNamesForParameter(type, name, value, index, count)
+    {
+        if (value === 0)
+            return null;
+
+        function test_and_clear_bit(names, value, bit, desc) {
```

Comment:
Style: please make this camelCase
Style: we normally don't abbreviate variable names, so we'd prefer `description` (or `name` since we're adding it to `names`)